apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/cert/ApolloCSR.java

```diff
@@ -26,14 +25,29 @@
  *
  * @author alukin@gmail.com
  */
-public class ApolloCSR extends CertBase{
+public class ApolloCSR extends CertBase {
 
     private static final Logger log = LoggerFactory.getLogger(ApolloCSR.class);
-
+    
+    public static ApolloCSR loadCSR(String path){
+        PKCS10CertificationRequest cr;
+        ApolloCSR res = null;
+        FileReader fr;
+        try {
+            fr = new FileReader(path);
```

Comment:
FileReader might be not closed here. Check that twice.

Reply:
fixed